client/packages/host/src/components/Sync/SyncModal.tsx

```diff
@@ -140,8 +140,8 @@ export const SyncModal = ({
           <Typography sx={{ paddingBottom: 2, fontSize: 12, maxWidth: 650 }}>
             {t('sync-info.summary')
               .split('\n')
-              .map(line => (
-                <div>{line}</div>
+              .map((line, index) => (
+                <div key={index}>{line}</div>
```

Comment:
Side Quest! Have added a key to this div to get rid of the error below

![Screenshot 2025-01-20 at 4 00 20 PM](https://github.com/user-attachments/assets/613284a5-8e43-4958-b970-5cdeaa1b5744)